content/es/load_balancer.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Balanceador de Carga
+status: Feedback Appreciated
+category: concept
+---
+
+## Qué es
+Un balanceador de carga es un método para distribuir el tráfico de red entrante a través de un grupo de servidores en el backend. La solución puede estar basada en software o hardware.
+
+## Problema que aborda
+El balanceador ayuda a resolver el problema referente con la alta disponibilidad y los sistemas distribuidos. Cuando se trabaja en una aplicación o servicio que necesita escalar a cientos de miles de usuarios, a menudo será necesario distribuir esa aplicación en varios servidores. El balanceador de carga es la "policía de tráfico" que enruta el tráfico.
+
+## Cómo ayuda
```

Comment:
There is a new suggestion for template headers in 2c9bc465434c15210917ab995cf0fdb33016ba58

```suggestion
## ¿Cómo ayuda?
```